edalize/icarus.py

```diff
@@ -16,7 +16,7 @@ class Icarus(Edatool):
 	iverilog -s$(TOPLEVEL) -c $(TARGET).scr -o $@ $(IVERILOG_OPTIONS)
 
 run: $(VPI_MODULES) $(TARGET)
-	vvp -n -M. -l icarus.log $(patsubst %.vpi,-m%,$(VPI_MODULES)) $(TARGET) -fst $(EXTRA_OPTIONS)
+	vvp -n $(VPP_OPTIONS) -M. -l icarus.log $(patsubst %.vpi,-m%,$(VPI_MODULES)) $(TARGET) -fst $(EXTRA_OPTIONS)
```

Comment:
Maybe better add `VPI_MODULES_DIR` after `-M` and use existing `VPI_MODULES`?

Reply:
I think that VPI_MODULES is used for modules you're building on the fly in Edalize, whereas the Cocotb one already exists externally. It would expect to be able to build and delete the library if I insert it in that list.